dll/win32/shell32/wine/shellpath.c

```diff
@@ -764,9 +763,7 @@ static BOOL PathMakeUniqueNameW(
     else
```

Comment:
Observation: All the terminating code at the end of the first if{} block: from line 745 (`LPWSTR pchTitle = pszDest + cchTitle;`) down to line 761 inclusive (closing bracket of the for-loop), is exactly the very same as the code at the end of the else{} block in lines 816 to line 832 inclusive. Thus, it appears you could factor out this common code and move it outside, at the end of the huge "if {...} else {...}" block.

This would however require to also move outside (before the "if {} else {}" block) the auxiliary variables `cchTitle`, `maxCount`.

Reply:
Fixed.